russian/close-eyes/close-eyes.md

```diff
@@ -6,4 +6,6 @@
 
 
 Через несколько часов вы зашли на кухню и обнаружили зефиринку на столе,
-а затем в ванне и на балконе...
\ No newline at end of file
+а затем в ванне и на балконе...
+
+It looked exactly like [these ones](http://uniconf.ru.images.1c-bitrix-cdn.ru/upload/iblock/c66/c66a51819fab7fd8baf5ba0f0ca97504.jpg?137163076386437).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.